ani-cli

```diff
@@ -167,7 +167,7 @@ decrypt_link() {
 	ajax=$(echo "$video_id" | openssl enc -aes256  -K "$secret_key" -iv "$iv" -a)
 
 	#send the request to the ajax url
-	curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569" | jq -r '.source[].file'
+	curl -s -H 'x-requested-with:XMLHttpRequest' "$ajax_url" -d "id=$ajax" -d "time=69420691337800813569"  | grep -Eo 'https?:\\\/\\\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)' | grep -oE "https:\\\/\\\/vidstreamingcdn.com.*&expiry=[0-9]+" | sed 's/\\//g' | sed '$d'
```

Comment:
Make this at least 5 lines

Reply:
replace vidstreaming with cdn.. because sometimes gogo servers gives the loadfast link instead of vidstreaming...